dev/com.ibm.ws.security.jwt_fat.builder/fat/src/com/ibm/ws/security/jwt/fat/builder/actions/JwtBuilderActions.java

```diff
@@ -70,6 +70,7 @@ public Page invokeJwtBuilder(String testName, LibertyServer server, String build
         Page response = null;
         try {
             response = invokeUrlWithParameters(testName, webClient, jwtBuilderUrl, requestParams);
+            webClient.close();
```

Comment:
It may be better to put this in a `finally` block. That way if the `invokeUrlWithParameters()` method throws an exception, we'll still close the `webClient`. Same comment goes for the other usages.